.github/workflows/brokenLinkCheck.yml

```diff
@@ -0,0 +1,35 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Broken Link Checker
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
```

Comment:
should there be a `pull_request` trigger too?

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
@hui1110 ping.